sdk/src/contracts/collections/features/chargebackWithdrawal.ts

```diff
@@ -0,0 +1,208 @@
+import { BigNumber, ContractTransaction, PopulatedTransaction } from 'ethers';
+import { asAddress, isSameAddress, ZERO_ADDRESS } from '../..';
+import { CollectionContract } from '../collections';
+import { SdkError, SdkErrorCode } from '../errors';
+import type { Signerish, TokenId, WriteOverrides } from '../types';
+import { FeatureFunctionsMap } from './feature-functions.gen';
+import { asCallableClass, ContractFunction } from './features';
+
+export type ChargebackWithdrawalArgs = {
+  tokenId: TokenId;
+  owner: string;
+  quantity: number;
```

Comment:
We can make them optional and require them for ERC1155
```suggestion
  owner?: Addressish;
  quantity?: number;
```